examples/09-nat-gateways.yaml

```diff
@@ -0,0 +1,18 @@
+# An example of ClusterConfig object for customizing NAT gateway setup:
+--- 
+apiVersion: eksctl.io/v1alpha5
+kind: ClusterConfig
+
+metadata:
+  name: cluster-2
```

Comment:
Please use `cluster-9`, per convention in other examples :)

Reply:
Sure